midend/simplifyBitwise.h

```diff
@@ -31,6 +31,16 @@ class SimplifyBitwise : public Transform {
 
  public:
     const IR::Node *preorder(IR::AssignmentStatement *as) override;
+    const IR::Node *preorder(IR::OpAssignmentStatement *as) override { return as; }
+    const IR::Node *preorder(IR::BAndAssign *as) override {
+        return preorder(static_cast<IR::AssignmentStatement *>(as));
+    }
+    const IR::Node *preorder(IR::BOrAssign *as) override {
+        return preorder(static_cast<IR::AssignmentStatement *>(as));
+    }
+    const IR::Node *preorder(IR::BXorAssign *as) override {
+        return preorder(static_cast<IR::AssignmentStatement *>(as));
+    }
```

Comment:
Maybe it is too late today, but it is not obvious to me why this is OK to transform the bitwise-op-assign just the same way as normal assignment. Can you please add some explanation? (the pass itself could also use an example)

Reply:
I'll expand the comment with more details on what the pass is doing, which will make it clear(er) why it works fine for bitwise and/or/xor op-assign as well.